arches/settings.py

```diff
@@ -67,6 +67,17 @@
 # Uncomment this line for a development setup after running the ubuntu_setup.sh script (do not use in production)
 # ELASTICSEARCH_CONNECTION_OPTIONS = {"timeout": 30, "verify_certs": False, "basic_auth": ("elastic", "E1asticSearchforArche5")}
 
+# If you need to connect to Elasticsearch via an API key instead of username/password, use the syntax below:
+# ELASTICSEARCH_CONNECTION_OPTIONS = {"timeout": 30, "verify_certs": False, "api_key": "<API_KEY>"}
```

Comment:
In the docs it looks like the "api_key" is a tuple not a string.  Can you verify this?
See: https://elasticsearch-py.readthedocs.io/en/v8.9.0/api.html#module-elasticsearch

Example from the docs
```
# Set 'api_key' on the constructor
client = Elasticsearch(
    "http://localhost:9200",
    api_key=("id", "api_key")
)
```

Reply:
Hi @apeters,

When you generate an API key, the following information is returned from Elasticsearch's `_security/api_key` API request:

``` json
{
    "id": "nuAyjYkBlSnQtat8wfiM",
    "name": "my-api-key",
    "api_key": "a9KzcSMmQampvQ6zj2JHFw",
    "encoded": "bnVBeWpZa0JsU25RdGF0OHdmaU06YTlLemNTTW1RYW1wdlE2emoySkhGdw=="
}
```

Instead of using the `id` and `api_key`, you can simply use the base64 `encoded` value on it's own. My reasoning is why store two secrets, when one will suffice. The `encoded` value is "{id}:{api_key}" base64 encoded, similarly to what happends you you are using basic authentication with username and password.

I can verify this approach works. Would you like the comments expanded to show both methods?

Reply:
@RobOatesHistoricEngland after taking a closer look at the docs, it looks like you're right.
From the docs `api_key: Union[str, Tuple[str, str], None] = None`.  It still might be nice to document both methods especially since they give the example of using both the `id` and the `key`.

Reply:
Have tested the tuple implementation and is working, as expected.